testnet/testnet-rewards.md

```diff
@@ -69,14 +69,124 @@ Validators earn $JOY allocations in the same way as for bounty contributions: a
-\
-`(TOTAL_COUNCIL_tJOY_REWARD/tJOY_BUDGET)*(NETWORK_PERFORMANCE_SCORE)^2`\
-\
-where `NETWORK_PERFORMANCE_SCORE`, and is in the interval \[0, 1]. This score, computed by Jsgenesis using public evaluation metrics, attempts to capture how well the network performed. The evaluation metrics will change from one council period to the next.
+Council Members earn $JOY a bit differently, specifically, the total $JOY amount shared equally among all members are reduced by what is called the _network peformance score,_ squared, as shown below:
+
+```
+TOTAL_COUNCIL_JOY_REWARD = JOY_BUDGET * (TOTAL_COUNCIL_tJOY_REWARD/tJOY_BUDGET) * NETWORK_PERFORMANCE_SCORE^2
+```
 
-Council members earn $JOY allocations: a share of the `JOY_BUDGET` equal to what share the earned $tJOY makes up of the `tJOY_BUDGET`.
+where `NETWORK_PERFORMANCE_SCORE`, and is in the interval \[0, 1]. This score, computed by Jsgenesis using public evaluation metrics, attempts to capture how well the network performed. The evaluation metrics will change from one council period to the next.
 
 ### Content Creator
 
 `<Coming>`
+
+
+## Scoring and Reward Example
+To clarify how the scoring and rewards works, here is straightforward example.
+Note that the inputs are somewhat random, so don't set your expectations based on them.
+
+### Council Period Parameters and weights
+| Name                  | Value                     |
+| --------------------- |:-------------------------:|
+| `JOY_BUDGET`          | 2,000,000/0.2%/USD120,000 |
+| `USD_SUBSIDY`         | 2,000                     |
+| `tJOY_BUDGET`         | 80,000,000                |
+| `COUNCIL_tJOY_REWARD` | 10,000,00                 |
+
+| Group                 | Weight                    |
+| --------------------- |:-------------------------:|
+| `B_W`                 | 5                         |
+| `C_W`                 | 3                         |
+| `D_W`                 | 6                         |
+| `F_W`                 | 1                         |
+| `HR_W`                | 5                         |
+| `M_W`                 | 2                         |
+| `P_W`                 | 2                         |
+| `SU_W`                | 2                         |
+| `LO_W`                | 1                         |
+| `SUM`                 | **27**                    |
+
+### Budget Allocation
+
+Of the tJOY 80M in the budget, we assume tJOY 78M was spent, as shown below:
+| Purpose         | Spending [tJOY] | Workers (ex Lead) | Lead [tJOY] | Workers [tJOY] |
+|-----------------|:---------------:|:-----------------:|:-----------:|:--------------:|
+| Builders        | 15              | 5                 | 3           | 2.4            |
+| Content         | 10.5            | 6                 | 1.8         | 1.45           |
+| Distributor     | 17              | 8                 | 3           | 1.75           |
```

Comment:
Probably a very stupid question.

What are the working groups spending tJOY on other than workers and lead?
Distributors for example?